### PR TITLE
fix(table header): right-aligned caret fix

### PR DESCRIPTION
**Pull request by FMorrison87**
This PR includes the new caret logic on the Contacts table and a solution for making the caret work when the column is right aligned, which is applied in the Jobs Table in Maintenance.

### CODE REVIEW

File: src/Components/MaintenancePanel/JobsTable/JobsTable.tsx [hunk 1]
```diff
@@ -79,11 +79,15 @@ const JobsTable = ({ jobs }: PropTypes) => {
 
   const handleTabeRowClick = (job: Job) => {
     dispatch({ type: 'EDIT_JOB', payload: job });
-    navigate('maintenance-details');
+    navigate('job-details');
```
Comment (NicDominguez) [inline src/Components/MaintenancePanel/JobsTable/JobsTable.tsx:82]: This was recently changed to maintenance-details please leave as was.

File: src/Components/MaintenancePanel/JobsTable/JobsTable.tsx [hunk 2]
```diff
@@ -79,11 +79,15 @@ const JobsTable = ({ jobs }: PropTypes) => {
...
   };
 
+  console.log(columnDefs);
```
Comment (NicDominguez) [inline src/Components/MaintenancePanel/JobsTable/JobsTable.tsx:85]: remove

**Review by NicDominguez — CHANGES_REQUESTED** [review]

File: src/Components/MaintenancePanel/JobsTable/JobsTable.tsx (hunk 1 above)
Comment (FMorrison87) [reply src/Components/MaintenancePanel/JobsTable/JobsTable.tsx:82]: My bad that must have been an overwrite from a merge.

Comment (FMorrison87) [reply src/Components/MaintenancePanel/JobsTable/JobsTable.tsx]: Fixed.

**Review by NicDominguez — APPROVED** [review]

**Pull request merged**